app/population/view/daily.njk

```diff
@@ -0,0 +1,140 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{% block pageTitle %}
+  {{ t(pageTitle) }} – {{ super() }}
+{% endblock %}
+
+{% block content %}
+  <header>
+    <h1 class="govuk-heading-xl">
+      {{ resultsAsDailySummary.date | formatDateWithDay }}
+    </h1>
+  </header>
+
+  {%  call appPanel({classes: 'govuk-grid-row govuk-!-margin-bottom-4 govuk-!-padding-bottom-0'}) %}
```

Comment:
You can drop the `govuk-grid-row` class as this is causing a double negative margin and the panel to sit outside of the content area.